docs/extensibility/form-section.md

```diff
@@ -283,14 +252,18 @@ KeyValuePair widgets render an `object` value as a list of fields. One is used f
 
 #### Example
 
-```json
-{
-  "path": "spec.my-data",
-  "widget": "KeyValuePair",
-  "description": "Key and value must start and end with an alphanumeric character.",
-  "tooltip": "Labels are intended to be used to specify identifying attributes of objects that are meaningful and relevant to users, but do not directly imply semantics to the core system.",
-  "keyEnum": ["prefix", "regex", "exact"]
-}
+```yaml
+path: spec.my-data
+widget: KeyValuePair
+description: Key and value must start and end with an alphanumeric character.
+tooltip:
+  Labels are intended to be used to specify identifying attributes of objects
+  that are meaningful and relevant to users, but do not directly imply semantics to
+  the core system.
```

Comment:
This is not a correct way to specify multi-line strings in yaml. either `>` or `|` has to be used
```suggestion
tooltip: >
  Labels are intended to be used to specify identifying attributes of objects
  that are meaningful and relevant to users, but do not directly imply semantics to
  the core system.
```

Reply:
Also I wonder about widget examples, as we've got a bit of an inconsistency here. Some examples are given as an array:
```
- source: spec.foo
  name: Foo
- source: spec.bar
  name: Bar
```
but most singhle-entry examples are given just as an object:
```
source: spec.foo
name: Foo
```
however not all.
Maybe for the sake of consistency we should use the same format, suggesting it's an element of an array:
```
- source: spec.foo
  name: Foo
```
even if it's just a single entry?